examples/keyword.rs

```diff
@@ -0,0 +1,20 @@
+/// Demostrates how to fetch and set keywords
+/// 
+/// Usage: cargo run --example keyword <keyword> <value>?
+/// Example: cargo run --example keyword decoration:rounding (prints value) 
+/// Example: cargo run --example keyword decoration:rounding  15 (sets value)
+
+use hyprland::keyword::Keyword;
+
+fn main() -> hyprland::Result<()> {
+    let args: Vec<_> = std::env::args().skip(1).collect();
+    let keyword = args[0].clone();
+    
+    match args.len() {
+        0 => panic!("You need to pass a keyword"),
+        1 => println!("{} value is {}", keyword, Keyword::get(&keyword)?.value),
+        _ => Keyword::set(keyword, args[1].clone())?
```

Comment:
i feel like this should be differently, because if the user provides more than 2 args, then they are just ignored, so either join the extra args, or throw an error for extra args. i would personally return an error tho

Reply:
Now panics with more than 2 args.